plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -9136,7 +9154,8 @@ protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMapp
     @Override
     protected TestTable createTableWithDefaultColumns()
     {
-        throw new SkipException("Hive connector does not support column default values");
+        abort("Hive connector does not support column default values");
+        throw new AssertionError(); // unreachable
```

Comment:
why do you follow this pattern? i.e. why is the `throw AssertionError` needed?

Reply:
the compiler needs to know about control flow